_posts/2017-07-28-reproducible-research-badges.md

```diff
@@ -0,0 +1,154 @@
+
+https://o2r.uni-muenster.de/api/1.0/badge/executable/10.1126%2Fscience.1092666
+
+This URL requests a badge for the reproducibility status of the paper “Global Air Quality and Pollution” from *[Science](http://science.sciencemag.org/)* magazine identified by the DOI [10.1126/science.1092666](https://doi.org/10.1126/science.1092666). When the request is sent, the following steps will happen:
+
+1. The badger tries to find a research paper inside the o2r platform, called Executable Research Compendium ([ERC](http://o2r.info/erc-spec/spec/)) with the given DOI
+2. If if finds an ERC, it looks for a matching *[job](http://o2r.info/o2r-web-api/job/)*, a report of a reproduction analysis.
+3. Depending on the reproduction result (`success`, `running`, or `failure`) specified in the job, the badger generates a green, yellow or red badge with matching text indicating the reproducibility of the specified research publication.
+4. The request is redirected to a [shields.io](https://shields.io/) URL link containing the color and textual information. If an extended badge is requested, the badger generates and sends a SVG graphic instead.
+
+The returned image contains the requested information, which is in this case a successful reproduction:
+
+https://img.shields.io/badge/executable-yes-44cc11.svg
+
+Badges for reproducibility, peer review status and license are color coded to provide additional visual aids indicating for example (un)successful reproducibility.
+
+While a common procedure is reused across the badge types, other badges are generated with different functions and get their information from other sources. The information for peer review and releasetime badges is requested from the external services DOAJ and Crossref. The spatial badge also uses the o2r services, but it additionally converts the spatial information from coordinates into textual information, i.e. place names, using the [Geonames API](http://www.geonames.org/export/web-services.html). 
+
+## Spread badges over the web
+
+So there is a great badge server, and databases providing manifold badge information, but how to get them displayed online? The sustainable way would be for research website operators to agree on a common badge system and design, and then incorporate these badges on their platforms. But we know that is never ever going to happen, so instead of waiting we created a chrome extension for common research websites. The o2r-extender (https://github.com/o2r-project/o2r-extender) automatically inserts badges into search results or publication pages using client-side browser scripting. Its available in the Chrome Web Store (*[here](https://chrome.google.com/webstore/detail/opening-reproducible-rese/fhhfncpkfohlhphlcgpkbpialfhkmbil)*)  and ready to be tried out.
```

Comment:
"Chrome" instead of "chrome", it is a product name. Also add a link to general information about Chrome extensions (e.g. to the word "extension").